packages/flutter_tools/test/commands.shard/hermetic/downgrade_test.dart

```diff
@@ -43,7 +43,7 @@ void main() {
   });
 
   testUsingContext('Downgrade exits on unknown channel', () async {
-    fileSystem.currentDirectory.childFile('.flutter_tool_state')
+    fileSystem.currentDirectory.childFile('tool_state')
```

Comment:
Should this file name have stayed the same?

Reply:
I changed `Config.test` to prepend the `.flutter_` prefix, so this can stay the same.